Godeps/_workspace/src/github.com/openshift/openshift-sdn/ovssubnet/api/types.go

```diff
@@ -31,21 +31,26 @@ type SubnetRegistry interface {
 	GetNetNamespace(name string) (NetNamespace, error)
 	WriteNetNamespace(name string, id uint) error
 	DeleteNetNamespace(name string) error
+
+	GetServicesNetwork() (string, error)
+	GetServices() (*[]Service, error)
+	WatchServices(receiver chan *ServiceEvent, stop chan bool) error
 }
 
 type SubnetEvent struct {
-	Type   EventType
-	Minion string
-	Sub    Subnet
+	Type EventType
+	Node string
```

Comment:
does this rename affect the serialization/deserialization? if so, how do we do a rolling update where some nodes are old and some are new?